lib/uff_db_loader/templates/uff_db_loader_initializer.erb

```diff
@@ -15,5 +15,6 @@ if defined?(UffDbLoader)
     # config.dumps_directory = '/path/to/dumps' # Defaults to Rails.root.join('dumps')
     # config.database_config_file = 'path/to/database.yml' # Defaults to Rails.root.join('config', 'database.yml')
     # config.container_name = ->(app_name, environment) { "#{app_name}_#{environment}_custom_suffix" } # Defaults to "#{app_name}_#{environment}_db". It accepts a static string too.
+    # config.local_restore_command_path = '' # Sets the path to the db-cli (pg_restore, mysql). Defaults to nil so it uses what is configured in $PATH.
```

Comment:
`Defaults to nil so it's using the default binary (mysql, pg_dump) in $PATH.` sounds a little bit more like what happens.

Reply:
Ah. Sorry. I missed this.

Reply:
See https://github.com/rmehner/uff_db_loader/pull/71